### ISSUE TITLE
PervMom support

### ISSUE THREAD

**Issue by insickin** [PAhelper/PhoenixAdult.bundle#158]
Thanks for adding PervMom support, while adding file's I came across: "PervMom - Say Yes To Undress" (https://www.pervmom.com/941/say-yes-to-undress/helena-price/part-1), the agent recognized it with: "PervMom - /941/say-yes-to-undress/helena-price/part-1" but it fills no metadata! All of my other scenes from PervMom are loaded correctly all but this one, can you take a look at this?

I've added a log:

[com.plexapp.agents.phoenixadult.log](https://github.com/PAhelper/PhoenixAdult.bundle/files/3085765/com.plexapp.agents.phoenixadult.log)

**Comment by kamuk90** [comment]
in sitePervMom.py change this on 41 line

```
    # Actors
    movieActors.clearActors()
    actors = detailsPageElements.xpath('//span[@class="p-small"]')[0].text_content().replace('Starring:','').strip().split(" and ")
    if len(actors) > 0:
        for actor in actors:
            actorName = actor
            posterUrl = detailsPageElements.xpath('//img[@class="img-fluid scene-trailer"]')[0].get("src")
            actorPhotoURL = posterUrl.replace('trailer_tour',posterUrl.split('/')[8])
            Log("posterUrl: " + posterUrl)
            Log("actorPhoto: " + actorPhotoURL)
            movieActors.addActor(actorName,actorPhotoURL)

    # Posters/Background
    try:
        art.append(detailsPageElements.xpath('//img[@class="img-fluid scene-trailer"]')[0].get("src"))
    except:
        pass
```

@claygoldfinch 
dude can you do a new bugfix to PervMom, I don't know how I can do that on GitHub (I'm a new user)

**Comment by claygoldfinch** [comment]
@insickin Fixed this issue in PR#203.
